README.md

```diff
@@ -1,8 +1,57 @@
-# nc-games project
+# NC-Games Project
 
 Link to hosted version:
 https://boardgame-reviews.cyclic.app/api
```

Comment:
I can't tell if this isn't showing as a link because of the format, but I'd recommend making it one if not so that people viewing this repo can just click on it directly.